drf_extra_fields/fields.py

```diff
@@ -82,6 +82,12 @@ def to_representation(self, file):
             try:
```

Comment:
Would it be possible to make this check using the `file`? If the file is null, then we can assume that the field is empty. It can be faster for remote storage backends (e.g. S3 Storage Backend)

Reply:
I'll give that a try and see if it still solves the problem we were facing.